### PR TITLE
Feature rak13006 support

### PR DESCRIPTION
**Pull request by kkcloudy**

### CODE REVIEW

File: interface/rak13006/RAK13006-MCP2518/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 RAKwireless
```
Comment (xoseperez) [inline interface/rak13006/RAK13006-MCP2518/LICENSE]: Should be 2022

**Review by xoseperez — CHANGES_REQUESTED** [review]
If the MCP2518 library is already published somewhere else we should not add it here again, this come will become obsolete as soon as the original repo gets updated,

File: interface/rak13006/RAK13006-MCP2518/LICENSE (hunk 1 above)
Comment (fox3344) [reply interface/rak13006/RAK13006-MCP2518/LICENSE]: @xoseperez yes, the [RAK13006-MCP2518](https://github.com/RAKWireless/RAK13006-MCP2518) library is published on the RAKWireless repo, so I need to remove this library and tell the user they can download the latest library from RAKWireless repo in the README.md file，right?

Comment (xoseperez) [reply interface/rak13006/RAK13006-MCP2518/LICENSE]: Right

**Pull request closed without merging**